Samples/BackgroundBlur/BackgroundBlur/BackgroundBlur.cpp

```diff
@@ -0,0 +1,515 @@
+// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
+// PARTICULAR PURPOSE.
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I believe all samples are under the MIT license

Reply:
This is leftover from the sample I built out from. I can delete and have the code under our MIT license.